src/Symfony/Component/Notifier/composer.json

```diff
@@ -17,7 +17,8 @@
     ],
     "require": {
         "php": ">=7.2.5",
-        "symfony/polyfill-php80": "^1.15"
+        "symfony/polyfill-php80": "^1.15",
+        "psr/log": "~1.0"
```

Comment:
I recognized this was missing, because the `EmailMessageTest` failed locally while using the Notification class that depends on psr/log.